core/src/main/java/io/questdb/cutlass/http/processors/JsonQueryProcessor.java

```diff
@@ -128,6 +128,7 @@ public JsonQueryProcessor(
             this.queryExecutors.extendAndSet(CompiledQuery.CREATE_USER, sendConfirmation);
             this.queryExecutors.extendAndSet(CompiledQuery.ALTER_USER, sendConfirmation);
             this.queryExecutors.extendAndSet(CompiledQuery.CANCEL_QUERY, sendConfirmation);
+            this.queryExecutors.extendAndSet(CompiledQuery.EMPTY, sendConfirmation);
```

Comment:
Please add a test with empty query that uses this context, e.g. in `IODispatcherTest`

Reply:
`sendConfirmation` sends `{"ddl": "OK"}`, which isn't appropriate for this (as well as many other cases that use it, like `COMMIT`, `ROLLBACK` and more). I changed it to respond with 200 OK, but JSON says `{"error": "empty query" ...}`.